File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function($http){
		
		this.findAll = function(){
			return $http.get('../brand/findAll.do');
		};
		
		this.findPage = function(page,rows){
			return $http.get('../brand/findPage.do?page=' + page + '&rows=' + rows);
		};
		
		this.add = function(entity){
			return $http.post('../brand/add.do', entity);
		};
		
		this.update = function(entity){
			return $http.post('../brand/update.do', entity);
		};
		
		this.findOne = function(id){
			return $http.post('../brand/findOne.do?id=' + id);
		};
		
		this.dele = function(ids){
			return $http.get('../brand/delete.do?ids=' + ids);
		};
		
		this.search = function(page,rows,searchEntity){
			return $http.post('../brand/search.do?page=' + page + '&rows=' + rows,searchEntity);
		};
		
		//下拉列表数据
		this.selectOptionList=function(){
			return $http.get('../brand/selectOptionList.do');
		}
	
	});

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;

/**
 * 品牌接口
 * @author Administrator
 *
 */
public interface BrandService {

	/**
	 * 返回所有的数据
	 * @return
	 */
	public List<TbBrand> findAll();
	
	/**
	 * 返回分页列表
	 * @param pageNum
	 * @param pageSize
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	/**
	 * 增加
	 * @param brand
	 */
	public void add(TbBrand brand);
	
	/**
	 * 查找指定Id的品牌
	 * @param id
	 * @return
	 */
	public TbBrand findOne(Long id);
	
	/**
	 * 修改
	 * @param brand
	 */
	public void update(TbBrand brand);
	
	/**
	 * 删除品牌
	 * @param ids
	 */
	public void delete(Long [] ids);
	
	/**
	 * 条件查询
	 * @param brand
	 * @param pageNum
	 * @param pageSize
	 * @return
	 */
	public PageResult findPage(TbBrand brand, int pageNum,int pageSize);
	
	/**
	 * 品牌下拉框数据
	 * @return
	 */
	public List<Map> selectOptionList();
	
}
